app/routes/orders/new.js

```diff
@@ -29,11 +30,13 @@ export default class NewRoute extends Route {
     });
 
     const eventDetails = await order.query('event', { include: 'tax' });
-    return {
+
+    return hash({
       order,
-      event : eventDetails,
+      event      : eventDetails,
       tickets,
-      form  : await eventDetails.query('customForms', {
+      taxDetails : eventDetails.isTaxEnabled && eventDetails.get('tax', { cache: true, public: true }) || undefined,
+      form       : await eventDetails.query('customForms', {
```

Comment:
Remove await

Reply:
Yes sir removing this,  I forgot to do this